hello

```diff
@@ -0,0 +1,6 @@
+set -o errexit
+set -x
+mkdir -p out bin
+go build -o bin/hello cmd/hello/hello.go
+bin/hello
+open hello.png
```

Comment:
Lack of a trailing newline here can, with some shells, prevent this command from being executed.